Assets/MRTK/Providers/WindowsMixedReality/XRSDK/WindowsMixedRealityDeviceManager.cs

```diff
@@ -6,9 +6,14 @@
 using Microsoft.MixedReality.Toolkit.Windows.Utilities;
 using Microsoft.MixedReality.Toolkit.XRSDK.Input;
 using System;
+using System.Linq;
 using System.Collections.Generic;
 using UnityEngine;
 using UnityEngine.XR;
+using UnityEngine.XR.Management;
+#if XR_PLUGIN_WINDOWSMR
```

Comment:
Where does this scripting define come from?

Reply:
From Unity's Windows MR Plugin/Unity.XR.WindowsMixedReality.asmdef.

Reply:
As far as I can tell, that define is specific to Unity's assembly and doesn't propagate over to other asmdefs. We have one defined for this assembly already though! `WMR_ENABLED`

Also, I'm seeing a build issue with referencing `WindowsMRLoader`:

![image](https://user-images.githubusercontent.com/3580640/108121973-75293580-7058-11eb-82be-fa3cb8ee8ac7.png)

Reply:
I'm going to take a pass on this PR to get it ready for 2.6

Reply:
@keveleigh Fixed it in my fork branch.

Reply:
The build issue referenced above is still present :( i'll open a PR to fix